pygments/lexers/php.py

```diff
@@ -85,6 +87,56 @@ class ZephirLexer(RegexLexer):
     }
 
 
+class PsyshConsoleLexer(Lexer):
+    """
+    For `PsySH`_ console output, such as:
+
+    .. sourcecode:: psysh
+
+        >>> $greeting = function($name): string {
+        ...     return "Hello, {$name}";
+        ... };
+        => Closure($name): string {#2371 …3}
+        >>> $greeting('World')
+        => "Hello, World"
+
+    .. _PsySH: https://psysh.org/
```

Comment:
Please add a `.. versionadded:: 2.7` note as well.

Reply:
I've made the requested addition. Sorry it took so long to get to. 😁